rfcs/1478-variables.md

```diff
@@ -0,0 +1,123 @@
+# Variables
+
+Assign values to names that can be referenced in Seq searches and queries.
+
+## Motivation
+
+Every time I go hunting a bug - one that isn't immediately obvious thanks to some blaring exception, error message, or stack trace - my desk ends up covered in scribbled notes, and my browser window filled with open tags.
+
+Seq should help organize more of this information, and bear a bit more of the cognitive load. Debugging production issues is difficult and stressful enough, at times, so the more Seq can do to help, the better.
+
+On the client-side, when I'm poking at an issue in a web page, I often assign little snippets of information to variables in the developer tools console, or by clicking an element of interest and assigning its DOM node to a variable that I can poke at in JavaScript.
+
+On the server-side, life would be made easier if Seq could offer a similar experience:
```

Comment:
Client-side / server-side here is confusing, because both the browser developer tools and Seq variables happen on the client. 

Can just delete 'On the client-side' and 'On the server-side'